doc/api/esm.md

```diff
@@ -2,6 +2,27 @@
 
 <!--introduced_in=v8.5.0-->
 <!-- type=misc -->
+<!-- YAML
+added: v8.5.0
+changes:
+  - version: v12.0.0
+    pr-url: https://github.com/nodejs/node/pull/26745
+    description:
+      Add support for ES modules using `.js` file extension via `package.json`
+      `"type"` field.
+  - version:
+    - v12.17.0
+    - v13.2.0
+    pr-url: https://github.com/nodejs/node/pull/29866
+    description: Unflag `--experimental-modules`.
```

Comment:
```suggestion
    description: Unflag ECMAScript modules.
```

...of...

```suggestion
    description: Unflag ESM.
```

...or...

```suggestion
    description: Loading ECMAScript modules no longer requires a command-line flag.
```

...or...

```suggestion
    description: Using the `--experimental-modules` is no longer needed.
```

Basically, you can unflag a feature, or you can remove a flag, but unflagging a flag seems strange.

That said, the meaning is clear enough, so this is not blocking if my other suggestions have different problems that we wish to avoid.